workspace/welcome-to-xod/305-document-it/patch.xodp

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Probably, we should tell somewhere in this tutorial about publishing library (but User should not publish this tutorial). What do you think? :)